src/storage/v2/property_value.hpp

```diff
@@ -330,6 +330,8 @@ inline bool operator<(const PropertyValue &first, const PropertyValue &second) n
   }
 }
 
+inline bool operator>(const PropertyValue &first, const PropertyValue &second) noexcept { return second < first; }
```

Comment:
clang say this may throw exception and it shouldn't throw. What to do about that?